node/src/actors/chain_manager/mod.rs

```diff
@@ -230,6 +230,8 @@ pub struct ChainManager {
     last_superblock_consensus: Option<CheckpointBeacon>,
     /// Settings for Threshold Activation of Protocol Improvements
     tapi: Tapi,
+    /// Transaction priority engine
+    priority_engine: PriorityEngine,
```

Comment:
In the beginning I put it in the `ChainState`, to then realize that it wouldn't be a great thing because we would most likely need migrations.

Then, I moved it to `ChainManager` and ran away from persisting the engine because:
- `PriorityEngine` is populated during synchronization, so fresh nodes and nodes that come back from being stopped for a while are in business immediately.
- There's a minimum number of blocks that need to be tracked before the estimator gives a positive result. Before that, the `priority` method provides insightful feedback telling you to wait X minutes.
- Rollbacks are rare and don't have a significant impact on priority (unless there's a mechanism for re-pushing rolled back transactions into the mempool, I guess?)